tests/common/snappi/common_helpers.py

```diff
@@ -550,89 +617,124 @@ def get_pfcwd_config_attr(host_ans, config_scope, attr):
     return None
 
 
-def get_pfcwd_poll_interval(host_ans):
+def get_pfcwd_poll_interval(host_ans, asic_value='None'):
```

Comment:
asic_value=None

Reply:
its actually 'None' because get_multidut_snappi_ports appends asic_value to 'None'